draft-ietf-cose-rfc8152bis-algs.xml

```diff
@@ -1677,23 +1682,33 @@ COSE_KDF_Context = [
         <name>Assignments for Existing Algorithms</name>
         <t>
           For the current set of algorithms in the registry, those in this document as well as those in <xref target="RFC8230"/> and <xref target="I-D.ietf-cose-hash-sig"/>, the capabilities is set to the single entry of the key type that will be accepted.
```

Comment:
"the capabilities is set to the single entry of the key type that will be accepted."

I know this is quite obvious from the previous section, but I'd really like a sentence stating what is the key type for the algorithm, because that is not part of the COSE Algorithms registry right now. A reference to the previous section would be enough, for example. 

"The key type values covering all algorithms at the time of writing this specification can be found in section 8.1"

or something of the sort.